namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-dev/resources/rds.tf

```diff
@@ -37,3 +37,37 @@ resource "kubernetes_secret" "court_case_rds" {
   }
 }
 
+module "court_case_service_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.0"
```

Comment:
The latest release of the RDS module is 5.1  Please update the source accordingly.

Reply:
Done